README.md

```diff
@@ -23,35 +23,50 @@ The jsep built files will be in the build/ directory.
 
 #### Client-side
 
-```javascript
-<script src="/PATH/TO/jsep.min.js"></script>
-...
-let parse_tree = jsep("1 + 1");
+```html
+<script type="module">
+import { Jsep } from '/PATH/TO/jsep.min.js';
+const parsed = Jsep.parse('1 + 1');
+</script>
 ```
 
 #### Node.JS
 
 First, run `npm install jsep`. Then, in your source file:
 
 ```javascript
-let jsep = require("jsep");
-let parse_tree = jsep("1 + 1");
+// ESM:
+import { Jsep } from 'jsep';
+const parse_tree = Jsep.parse('1 + 1');
```

Comment:
What is the advantage of this for our users over just using `jsep()`?

Reply:
I'll go back to both. I've been reading tonight a lot about the "evils" of default exports ... so I think our preferred usage should be to use the named export, but I've got it working for both CJS and ESM with both the default and named export now (without needing '.default')

Reply:
I always try to keep an open mind, so feel free to send these "evils" my way, but so far I'm entirely unconvinced by the arguments against default exports. The ones I've seen seem to mainly center around inadequacies of tooling. IMHO it's the tooling that needs to be fixed, not default exports that should be avoided. There is a **very** clear usability benefit to default exports. Not only can the module be imported without looking up its documentation to remember what name to use, but also it offers a clear entry point, that *this* object is the main thing to use. And the syntax is simpler too. These may seem trivial for experienced developers, but they can make a world of difference to beginners who get easily discouraged and frustrated.

Reply:
I was waiting for a response to this, not sure if it still applies?

Reply:
ah sorry! I thought it was resolved by providing both named and default exports?
From what I understand:
- using default imports prevents tree-shaking if there are parts of a file that are unused
- mixing named and default exports seems to be discouraged (which is why the build is more difficult for both cjs and esm). But I like that it allows both use cases.
- default imports can lead to inconsistent naming in a code base and may make large-scale refactoring more difficult.
- intellisense may be more problematic with default imports (tooling problem, I know)
- named exports are easier in export aggregation (i.e. barrel files)

In my work, I use a lot of both CJS default and named require statements, but mostly named imports in typescript. (Trying to get away from import * as _ from 'lodash'!). I've found that unit testing named exports can sometimes be easier. But overall, I don't have a strong opinion either way.